frontend/packages/dev-console/src/components/import/section/BuildSection.tsx

```diff
@@ -42,6 +42,10 @@ export const BuildSection: React.FC<BuildSectionProps> = ({ values, appResources
       };
       gitService && checkTektonFolder();
     }
+  }, 2000);
+
+  React.useEffect(() => {
+    autoSelectPipelines();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [values?.git?.url, isRepositoryEnabled, setFieldValue]);
```

Comment:
Please cross-check whether `autoSelectPipelines` should be added to the dependency array.
(Errors are suppressed for this currently due to the eslint comment)